internal/config/config_test.go

```diff
@@ -297,7 +303,11 @@ func TestGetGVKs(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			gvks, err := tc.scanType.GetGVKs(fakeDiscovery, fakeLog)
 			require.NoError(t, err)
-			require.ElementsMatch(t, tc.expectedGVKs, gvks)
+			expectedGVKs := make([]GroupVersionKind, len(tc.expectedGVKs))
+			for i, expectedGVK := range tc.expectedGVKs {
+				expectedGVKs[i] = GroupVersionKind{GroupVersionKind: expectedGVK, PreferredVersion: "v1"}
```

Comment:
what stops us from doing the same thing as in the other change in this file, where we've simply switched the `expectedGVK` type to a `config.GroupVersionKind`? 🤔

Reply:
Nothing except my laziness 😂. I'll change it

Reply:
Rebased on master too, reflected this change onto helm_test.go